Goldmine/Gold.java

```diff
@@ -0,0 +1,71 @@
+
+
+import java.util.*; 
+  
+class Gold{ 
```

Comment:
Either keep all opening braces in next line or all braces next to statements separated by space. Please maintain uniformity throughout code.